system/greenhouse-ccloud/templates/kube-monitoring-pluginpreset.yaml

```
@@ -125,6 +125,49 @@ spec:
       value: thanos.yaml
   {{- end }}
   {{- end }}
+  {{- range $cluster, $additionalSettings := .Values.kubeMonitoring.controlplaneClusters }}
+  - clusterName: {{ $cluster }}
+    overrides:
+    - name: kubeMonitoring.crds.enabled
+      value: false
+    - name: kubeMonitoring.crds.prometheusOperator.enabled
```

```suggestion
    - name: kubeMonitoring.prometheusOperator.enabled
```
ref: https://github.com/cloudoperators/greenhouse-extensions/blob/9982b543d3f73f52a39f56c384c0b1d7e4e6f26c/kube-monitoring/charts/values.yaml#L105C3-L108